base/client.jl

```diff
@@ -210,7 +210,19 @@ end
 
 # call include() on a file, ignoring if not found
 include_ifexists(mod::Module, path::AbstractString) = isfile(path) && include(mod, path)
-
+function ttyhascolor()
+    term_type = get(ENV, "TERM","")
+    startswith(term_type, "xterm") && return true
+    try
+        @static if Sys.KERNEL === :FreeBSD
+            return success("tput AF 0")
```

Comment:
```suggestion
            return success(`tput AF 0`)
```

Reply:
lol I didn't even notice this; the fact that it works at all is because this is inside of a `try` :smile: